persistence/persistence-api/src/main/java/org/cbioportal/persistence/GenePanelRepository.java

```diff
@@ -42,7 +42,7 @@
 
 public interface GenePanelRepository {
     
-        String getGenePanelBySampleIdAndProfileId(String sampleId, String profileId);
```

Comment:
This is not needed. We are able to get the samples in a profile, and for now the only use case is given a profile and set of genes, what are the sequenced genes.